src/App.js

```diff
@@ -237,7 +237,7 @@ class App extends React.Component {
     if (!mode) return null
 
     return (
-      <AragonApp publicUrl="./aragon-ui/">
+      <AragonApp publicUrl="/aragon-ui/">
```

Comment:
Accidental revert of https://github.com/aragon/aragon/pull/373?

Reply:
Awkward, I meant to revert the other commit but ended up accidentally reverting the one before it. Sorry!